utils/dc-chain/Makefile.default.cfg

```diff
@@ -22,7 +25,7 @@
 # - gccrs-dev:    GCC fork for development of the GCCRS Rust compiler.
 # - rustc-dev:    GCC fork for development of the libgccjit rustc GCC codegen.
 # If unsure, select stable. See README.md for more detailed descriptions.
-toolchain_profile=stable
+toolchain_profile=rustc-dev
```

Comment:
Should these and the other settings have changed as well? I'm thinking you accidentally put up your own Makefile.cfg that you were testing with?

Reply:
Yeah, I force-pushed the defaults back. 😅